services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -224,6 +224,18 @@ def compose_spec_submitted(self) -> bool:
         ),
     )
 
+    skip_sidecar_monitor_and_removal: bool = Field(
```

Comment:
IMO ``DynamicSidecar(BaseModel)`` is becoming a monolithic with a flat list of unrelated options. It mixes 
- mutable/unmutable state fields
- some are set upon init and some using assignment (and only the first validated)
- unrelated topics a

What about splitting and applying separation of concerns?